packages/shared/src/components/post/PostComments.tsx

```diff
@@ -95,7 +95,7 @@ export function PostComments({
   }
 
   return (
-    <div className="mb-12 flex flex-col gap-4" ref={container}>
+    <div className="mb-12 mt-6 flex flex-col gap-4" ref={container}>
```

Comment:
In case of mobile, we do not show `NewComment` button on the post page anymore. Need to preserve the gap here.

Changed the `NewComment` styles accordingly, so that desktop renders without a change.